src/Public/LogRhythm/Case/Get-LrCasePlaybookProcedures.ps1

```diff
@@ -0,0 +1,155 @@
+using namespace System
+using namespace System.IO
+using namespace System.Collections.Generic
+
+Function Get-LrCasePlaybookProcedures {
+    <#
+    .SYNOPSIS
+        Return a list of procedures on a playbook on a case.
+    .DESCRIPTION
+        The Get-LrCasePlaybookProcedures cmdlet returns a list of procedures associated
+        with a playbook that has been assigned to a specific case.
+
+        If no PlaybookID is specified the first playbook assigned to the case will be returned.
+    .PARAMETER Credential
+        PSCredential containing an API Token in the Password field.
+        Note: You can bypass the need to provide a Credential by setting
+        the preference variable $SrfPreferences.LrDeployment.LrApiToken
+        with a valid Api Token.
+    .PARAMETER CaseId
+        Unique identifier for the case, either as an RFC 4122 formatted string, or as a number.
+    .PARAMETER PlaybookId
+        (Optional) Unique identifier for the playbook, either as an RFC 4122 formatted string, or as a number.
+    .INPUTS
+        [System.Object]   ->  CaseId
+        [System.String]   ->  PlaybookId
+    .OUTPUTS
+        System.Object representing the returned LogRhythm playbook procedures on the applicable case.
+
+        If no prceodures are found, this cmdlet will return null.
+    .EXAMPLE
+        PS C:\> Get-LrCasePlaybookProcedures -Credential $Token -CaseId 8703 -PlaybookId "4CAB940D-CFF7-442E-A54A-5D4949FA783D"
+        ---
+        id            : C8C47BEC-7E77-44C0-AB7A-3DFA2AF6E9FF
+        name          : Drill down on the alarm to gain additional insight
+        description   :
+        assignee      :
+        status        : NotCompleted
+        dueDate       :
+        notes         :
+        dateUpdated   : 2019-12-23T13:36:04.3544575Z
+        lastUpdatedBy : @{number=227; name=Domo, Derby; disabled=False}
+
+        id            : 1900E73A-B1C2-4C76-95C4-5E251C7E3BC6
+        name          : Determine if the event is an incident
+        description   :
+        assignee      :
+        status        : NotCompleted
+        dueDate       :
+        notes         :
+        dateUpdated   : 2019-12-23T13:36:04.3544575Z
+        lastUpdatedBy : @{number=227; name=Domo, Derby; disabled=False}
+    .NOTES
+        LogRhythm-API
+    .LINK
+        https://github.com/SmartResponse-Framework/SmartResponse.Framework        
+    #>
+
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory = $false, Position = 0)]
+        [ValidateNotNull()]
+        [pscredential] $Credential = $SrfPreferences.LrDeployment.LrApiToken,
+
+
+        [Parameter(
+            Mandatory = $true,
+            ValueFromPipeline = $true,
+            Position = 1
+        )]
+        [ValidateNotNullOrEmpty()]
+        [object] $CaseId,
+
+        [Parameter(
+            Mandatory = $false,
+            ValueFromPipeline = $true,
```

Comment:
Change `ValueFromPipeline` on the `$PlaybookId` parameter to `ValueFromPipelineByPropertyName` and change the name of the parameter to `Id`, so that it matches the output of the Playbook object.

The idea is to allow someone to do the following:

```powershell
PS> Get-LrCasePlaybooks -Id 2068 | Get-LrCasePlaybookProcedures -CaseId 2068
```

Unfortunately the Playbook objects returned by the API for CasePlaybooks does not include the CaseId, so it has to be specified twice.  :(

Reply:
Done, for this and for Update-LrCasePlaybookProcedures